brut.apktool/apktool-lib/src/main/java/brut/androlib/res/AndrolibResources.java

```diff
@@ -457,6 +572,58 @@ public void aaptPackage(File apkFile, File manifest, File resDir, File rawDir, F
         }
     }
 
+    public void aaptPackage(File apkFile, File manifest, File resDir, File rawDir, File assetDir, File[] include)
+            throws AndrolibException {
+
+        boolean customAapt = false;
+        String aaptPath = apkOptions.aaptPath;
+        List<String> cmd = new ArrayList<String>();
+
+        // path for aapt binary
+        if (! aaptPath.isEmpty()) {
+            File aaptFile = new File(aaptPath);
+            if (aaptFile.canRead() && aaptFile.exists()) {
+                aaptFile.setExecutable(true);
+                cmd.add(aaptFile.getPath());
+                customAapt = true;
+
+                if (apkOptions.verbose) {
+                    LOGGER.info(aaptFile.getPath() + " being used as aapt location.");
+                }
+            } else {
+                LOGGER.warning("aapt location could not be found. Defaulting back to default");
+
+                try {
+                    cmd.add(getAaptBinaryFile().getAbsolutePath());
+                } catch (BrutException ignored) {
+                    cmd.add("aapt");
```

Comment:
This is hardcoded to aapt1, if the system is in aapt2 and the aapt2 binary couldn't be loaded, this would execute aapt1